lighthouse-core/test/gather/gatherers/trace-elements-test.js

```diff
@@ -741,3 +680,77 @@ describe('Trace Elements gatherer - Animated Elements', () => {
     ]);
   });
 });
+
+describe('instrumentation', () => {
+  it('resolves animation name', async () => {
+    const connectionStub = new Connection();
+    connectionStub.on = createMockOnFn()
+      .mockEvent('protocolevent', {
+        method: 'Animation.animationStarted',
+        params: {animation: {id: '1', name: 'example'}},
+      });
+    connectionStub.sendCommand = createMockSendCommandFn()
+      .mockResponse('Animation.enable')
+      .mockResponse('Animation.disable');
+    const driver = new Driver(connectionStub);
+    const gatherer = new TraceElementsGatherer();
+    await gatherer.startInstrumentation({driver});
+
+    // Flush events.
+    jest.advanceTimersByTime(1);
```

Comment:
fyi we also have `flushAllTimersAndMicrotasks` in `test-utils.js` for this purpose. this is a pretty simple case, but it's a handy one to use if there's something more complex in the future.